### PR TITLE
Content/guidance key - Anonymous slot

### PR DESCRIPTION
**Pull request by andij**

### CODE REVIEW

File: src/content/docs/components/ns-alert.mdx [hunk 1]
```diff
@@ -68,9 +68,9 @@ This component immediately brings the content to the user's attention. It can ta
 
 | Key | Field type | Guidelines |
 | :--- | :--- | :--- |
-| <span class="marker">A</span> | Icon | This is the icon for the `highlighter` type. It is dictated by the `state` of ns-alert. The [specification](#specification) has a list of options. |
+| <span class="marker">A</span> | Icon | The icon is dictated by the [`state`](#specification-attribute-state). |
 | <span class="marker">B</span> | Heading | The recommended length is between 4 and 12 words, not exceeding 50 characters in total. |
-| <span class="marker">C</span> | Content | A single, short paragraph works best. The copy length should not exceed more than two paragraphs of 3 lines each. Include bold copy `<b>`, inline links `<a>`, and a caveat at the end of the relevant paragraph if required `<a href="#caveat">1</a>`. |
+| <span class="marker">C</span> | Anonymous slot | A single, short paragraph works best. Copy length should not exceed two paragraphs of 3 lines. Include emphasis with `<strong>`, inline links `<a>`, lists `<ul>` `<ol>`. Add [caveat link](/components/ns-caveat) if required. |
```
Comment (RobTobias123) [inline src/content/docs/components/ns-alert.mdx:73]:
```suggestion
| <span class="marker">C</span> | Anonymous slot | A single, short paragraph works best. Text length should not exceed two paragraphs of 3 lines. Include emphasis with `<strong>`, inline links `<a>`, lists `<ul>` `<ol>`. Add a [caveat link](/components/ns-caveat) if required. |
```

File: src/content/docs/components/ns-pill.mdx [hunk 2]
```diff
@@ -64,7 +64,7 @@ Pills can help to create a subtle sense of urgency or to draw a customer’s att
 | Key | Field type | Guidelines |
 | :--- | :--- | :--- |
 | <span class="marker">A</span> | Icon | Use an icon to support your description. |
-| <span class="marker">B</span> | Content | Content should be very concise and catchy, ideally between 1 and 4 words. Your content should not run over one line on a `squash-court` viewport. |
+| <span class="marker">B</span> | Anonymous slot | Be concise and catchy, ideally between 1 and 4 words. Content should not run over one line on a `squash-court` viewport. |
```
Comment (RobTobias123) [inline src/content/docs/components/ns-pill.mdx:67]:
```suggestion
| <span class="marker">B</span> | Anonymous slot | Be concise and catchy, ideally between 1 and 4 words. Content should not run over more than one line on a `squash-court` viewport. |
```

**Review by RobTobias123 — COMMENTED** [review]
Couple of suggestions made

File: src/content/docs/components/ns-pill.mdx (hunk 2 above)
Comment (andij) [reply src/content/docs/components/ns-pill.mdx]: Done.

File: src/content/docs/components/ns-alert.mdx (hunk 1 above)
Comment (andij) [reply src/content/docs/components/ns-alert.mdx]: Done.

**Review by RobTobias123 — APPROVED** [review]
Looks good

**Pull request merged**